multiworld/envs/mujoco/__init__.py

```diff
@@ -0,0 +1,30 @@
+from gym.envs.registration import register
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+_REGISTERED = False
+
+
+def register_custom_envs():
+    global _REGISTERED
+    if _REGISTERED:
+        return
+    _REGISTERED = True
+
+    LOGGER.info("Registering multiworld mujoco gym environments")
+    # TODO
```

Comment:
why is this code all commented out? is this supposed to be an example? if so it would be good to maybe make that a bit more clear